os/board/common/crashdump.c

```diff
@@ -167,6 +311,35 @@ static int ramdump_via_uart(void)
  ****************************************************************************/
 
 #if defined(CONFIG_BOARD_CRASHDUMP)
+void crashdump_init(void)
+{
+	int index;
+
+	index = mtd_getpartitionindex("coredump");
+	if (index < 0) {
+		lldbg("partition index is wrong for coredump part name\n");
+		return;
+	}
+	snprintf(coredump_partname, MTD_PARTNAME_LEN + 1, "/dev/mtdblock%d", index);
+
+	coredump_partsize = mtd_getpartitionsize(index, &coredump_partoffset);
+	if (!coredump_partsize) {
+		lldbg("partition index is wrong for coredump part name\n");
```

Comment:
Isn't return needed for this case?

Reply:
we don't handle the return case here as its void function.